### DISCUSSION TITLE
SHA256 mismatch screaming-frog-seo-spider due to new version

### DISCUSSION THREAD

**Discussion by pat-flew** [Homebrew/discussions#1328] [Casks]
Seems they are redirecting download from version 14.1 to 15.1 and that is causing the mismatch.

Let me know if I should be reporting to the author instead!

brew install screaming-frog-seo-spider

`==> Downloading https://download.screamingfrog.co.uk/products/seo-spider/ScreamingFrogSEOSpider-14.1.dmg
==> Downloading from https://download.screamingfrog.co.uk/products/seo-spider/ScreamingFrogSEOSpider-15.1.dmg
######################################################################## 100.0%
Error: SHA256 mismatch
Expected: 57e94997351e2ddc369af475f7651cfaa1d0420ec96752a0a91ec6a2f77cc04f
  Actual: 6da4dd294c1879789ad097d3ea60dcda070b0fbce7db56cd19c38457b58c9963
    File: /Volumes/Data/Users/Pat/Library/Caches/Homebrew/downloads/58d27dcbfb37dd4f87713bd363f1d846943a9de3e788b05029c36e5707c8b9c5--ScreamingFrogSEOSpider-15.1.dmg`

**Comment by SMillerDev** [comment]
It's kind of weird to redirect downloads. But you could make a pull request to update the cask to fix it.

**Comment by chenrui333** [comment]
I think the root cause is they forgot to run `brew update` and it consumes the 14.1 release instead of 15.1 (looking at the previous PRs, did not find any checksum mismatch issue)

relates to:
- https://github.com/Homebrew/homebrew-cask/pull/100085
- https://github.com/Homebrew/homebrew-cask/pull/104021
